main.c

```diff
@@ -80,19 +80,19 @@ static void fatalError(const enum fatal_error type)
         taskDISABLE_INTERRUPTS();
 
         for(;;) {
-                LED_enable(1);
-                LED_enable(2);
+                led_enable(LED_GPS);
```

Comment:
if we have a fatal error we should be at least flashing the error LED as well. maybe flash all LEDs?